### ISSUE TITLE
draft-livingood-low-latency-deployment-03 edits and comments

### ISSUE THREAD

**Issue by gregbo** [jlivingood/IETF-L4S-Deployment#55]
Here are a few comments on the draft:

- Why is the draft called _**Comcast ISP** Low Latency Deployment Design Recommendations_?  Aside from the Lotus report, I didn't see anything in the draft that was specific to Comcast ISPs.
- Some of the language in §5.1 seems a bit strong.  For example, I'd rephrase "will always make mistakes" as "could make mistakes."
- A citation for the "majority of traffic is now encrypted" claim would strengthen it.  For example, a [2020 Fortinet study](https://www.fortinet.com/blog/industry-trends/keeping-up-with-performance-demands-of-encrypted-web-traffic) gives a figure of 85% web traffic.  A [2020 VMware study](https://blogs.vmware.com/security/2020/09/network-security-encrypted.html) gives a figure of 80-90% web traffic.

I've also attached a unified diff of the text version of the draft and my edits.

[draft-livingood-low-latency-deployment-03.diff.txt](https://github.com/jlivingood/IETF-L4S-Deployment/files/13198606/draft-livingood-low-latency-deployment-03.diff.txt)

**Comment by jlivingood** [comment]
> Why is the draft called Comcast ISP Low Latency Deployment Design Recommendations? Aside from the Lotus report, I didn't see anything in the draft that was specific to Comcast ISPs.

Ack. Updating that in -05

**Comment by jlivingood** [comment]
> Some of the language in §5.1 seems a bit strong. For example, I'd rephrase "will always make mistakes" as "could make mistakes."

Ack. Updating to soften.

**Comment by jlivingood** [comment]
> A citation for the "majority of traffic is now encrypted" claim would strengthen it. For example, a [2020 Fortinet study](https://www.fortinet.com/blog/industry-trends/keeping-up-with-performance-demands-of-encrypted-web-traffic) gives a figure of 85% web traffic. A [2020 VMware study](https://blogs.vmware.com/security/2020/09/network-security-encrypted.html) gives a figure of 80-90% web traffic.

Ack - will add cites

**Comment by jlivingood** [comment]
THANK YOU! Made changes from the diff file as well.
